airflow/www/static/js/dag/StatusBox.tsx

```diff
@@ -34,12 +35,28 @@ export const boxSizePx = `${boxSize}px`;
 
 interface StatusWithNotesProps extends BoxProps {
   state: TaskState;
+  tryNumber?: number;
   containsNotes?: boolean;
 }
 
+const displayedRetryIcon = (
+  tryNumber: number | undefined,
+  state: TaskState
+) => {
+  if (
+    typeof tryNumber === "number" &&
+    tryNumber > 1 &&
+    (state === "success" || state === "failed")
```

Comment:
Why do we care about checking state? It might be helpful to know that a running task has multiple tries too.

Reply:
Usually where this is useful, it's once all the tasks have completed, this would allow you to see if any had retried with a quick glance, but I guess it could be useful to see if any running tasks where being retried. 

It would look like this:

<img width="576" alt="image" src="https://github.com/user-attachments/assets/c98615f5-3da0-41cc-aa4f-cb00c2accde7">

and here in dark mode:

<img width="565" alt="image" src="https://github.com/user-attachments/assets/ff88051a-8d37-4bb2-ade8-420e01485acd">